pubspec.lock

```diff
@@ -28,7 +28,7 @@ packages:
       name: charcode
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.3.1"
+    version: "1.2.0"
```

Comment:
Is it a bad idea to upgrade?

Reply:
Newer versions, listed in 'Latest', may not be mutually compatible.